docs/GettingStarted.asciidoc

```diff
@@ -298,7 +298,12 @@ the index page.
 
 === Cleanup
 IMPORTANT: openQA automatically deletes data that it considers "old" based on
-different settings. For example job data is deleted from old jobs by the `gru` task.
+different settings. For example old jobs and assets are deleted at some point.
+
+*NOTE* Cleanup jobs run within the Minion job queue provided by
```

Comment:
Paragraph is good but does not fit well within the "getting started" guide which targets beginners, not admins. Probably better in "Installing"?

Reply:
I suppose that counts for the whole cleanup section.

Reply:
hm, wouldn't say so necessarily. I think it's good to explain the concept of cleanup. Doesn't mean we need to explain all configuration options at once.

Reply:
I created another section under "Advannced configuration" and linked it here.